examples/integration-tutorials/transforming-message-content/README.md

```diff
@@ -1 +1,145 @@
-# transforming-message-content
+# Message Transformation
+
+This guide demonstrates how to transform a request message payload to a different format expected by the back-end service.
```

Comment:
Since this is a continuation of an integration tutorial can we rephrase and update sections thinking that a user is going through this example in order?